apps/src/music/progress/MusicValidator.ts

```diff
@@ -47,26 +51,41 @@ export default class MusicValidator extends Validator {
         eventData.when + length > currentPlayheadPosition
       ) {
         currentNumberSounds++;
+
+        if (eventData.triggered) {
+          this.conditionsChecker.addSatisfiedCondition({
+            name: KnownConditionNamesList.PLAYED_SOUND_TRIGGERED,
+          });
+        }
       }
     });
 
     if (currentNumberSounds === 3) {
-      this.conditionsChecker.addSatisfiedCondition(
-        KnownConditions.PLAYED_THREE_SOUNDS_TOGETHER
-      );
+      this.conditionsChecker.addSatisfiedCondition({
+        name: KnownConditionNamesList.PLAYED_SOUNDS_TOGETHER,
+        value: 3,
+      });
     }
     if (currentNumberSounds === 2) {
-      this.conditionsChecker.addSatisfiedCondition(
-        KnownConditions.PLAYED_TWO_SOUNDS_TOGETHER
-      );
+      this.conditionsChecker.addSatisfiedCondition({
+        name: KnownConditionNamesList.PLAYED_SOUNDS_TOGETHER,
+        value: 2,
+      });
```

Comment:
Could we just use `currentNumberSounds` as `value` when calling `addSatisfiedCondition`?

Reply:
Good call.  I'll refactor this code some more.